tensorboard/pip_package/build_pip_package.sh

```diff
@@ -135,9 +115,66 @@ pip install -qU wheel 'setuptools>=36.2.0'
 python setup.py bdist_wheel --python-tag py2 >/dev/null
 python setup.py bdist_wheel --python-tag py3 >/dev/null
 
-if [ "$run_smoke_test" = 1 ]; then
-  smoke 2
-  smoke 3
-fi
+smoke() {
+  pymajorversion="$1"
+  if [ -z "${pymajorversion}" ]; then
+    pymajorversion="$(python -c 'import sys; print(sys.version_info[0])')"
+  fi
+  smokepython="python$1"
+  smokevenv="smoke-venv$1"
```

Comment:
As fun as it is to always parse these names as smock-ev-env and
smocket-eff (a new addition to the socket(2) family?), please add
underscores per style.

Reply:
Done.